components/more_channels/more_channels.scss

```diff
@@ -0,0 +1,270 @@
+            }
+        }
+    }
+
+    .modal-body {
+        padding: 15px 0 0;
+
+        .filtered-user-list {
+            height: calc(100vh - 300px);
+        }
+
+        .more-modal__row {
+            padding: 0 32px;
+            border-bottom: none;
+
+            .more-modal__details {
+                padding-left: 0;
+                color: rgba(var(--center-channel-color-rgb), 0.56);
+
+                svg {
+                    flex-shrink: 0;
+                }
+
+                .more-modal__name {
+                    align-items: center;
+                    margin-top: 0;
+
+                    span {
+                        color: var(--center-channel-color);
+                        font-weight: 500;
+                    }
+                }
+
+                #channelPurposeContainer {
+                    display: flex;
+                    align-items: center;
+                    justify-content: flex-start;
+
+                    .dot {
+                        width: 3px;
+                        height: 3px;
+                        flex-shrink: 0;
+                        background-color: rgba(var(--center-channel-color-rgb), 0.56);
+                        border-radius: 50%;
+                    }
+
+                    .more-modal__description {
+                        margin-left: 4px;
+                        font-weight: 400;
+                    }
+
+                    #membershipIndicatorContainer {
+                        display: flex;
+                        align-items: center;
+
+                        span,
+                        svg {
+                            color: var(--online-indicator);
+                        }
+                    }
+
+                    span {
+                        margin: 0 4px;
+                        font-size: 12px;
+                        font-weight: 600;
+                        line-height: 12px;
+                        opacity: 1;
+                    }
+                }
+            }
+
+            .more-modal__actions {
+                button {
+                    display: none;
+                    width: 54px;
+                    height: 32px;
+                    font-size: 12px;
+                    font-weight: 600;
+                }
+            }
+        }
+
+        .more-modal__row:hover {
+            background-color: rgba(var(--center-channel-color-rgb), 0.08);
+            cursor: pointer;
+
+            .more-modal__actions {
+                .primaryButton,
+                .outlineButton {
+                    display: inline-block;
+                }
+            }
+        }
+
+        .form-group {
+            padding: 0 32px;
+            margin-bottom: 0;
+        }
+
+        ::-webkit-scrollbar {
+            width: 4px;
+        }
+
+        ::-webkit-scrollbar-track {
+            background: none;
+        }
+    }
+
+    .modal-header {
+        .GenericModal__header {
+            width: 95%;
```

Comment:
add a small amount of padding to the right side of this so the `Create New Channel` button is not placed right against the `x` close button
```
padding-right: 4px
```